theme6/source/stylesheets/site.css.scss

```diff
@@ -1,24 +1,184 @@
-body {
-  background-color: #fbc547;
-  color: #333;
-  font-family: -apple-system, BlinkMacSystemFont, "Avenir Next", "Avenir",
-    "Segoe UI", "Lucida Grande", "Helvetica Neue", "Helvetica", "Fira Sans",
-    "Roboto", "Noto", "Droid Sans", "Cantarell", "Oxygen", "Ubuntu",
-    "Franklin Gothic Medium", "Century Gothic", "Liberation Sans", sans-serif;
-  padding: 18vh 1rem;
+/*==========Stylesheet DESKTOP========*/
+footer {
```

Comment:
I'd extremely suggest to add a class to the footer of the whole page because there may be multiple `footer` tags between sections and not only one for the document, if there's a section with a footer it is going to have the same styles as the footer of the document